config.yml

```diff
@@ -49,7 +49,22 @@ maxAuthAttempts: 3
 logInvalidAuthData: true
 maxMessageSize: 1048576
 
+#Authentication
+authType: open
+
+# authType: file
+# path: ./users.json #path to the user file
+# hash: 'md5' #the name of a HMAC digest algorithm
+# iterations: 100 #the number of times the algorithm should be applied
+# keyLength: 32 #the length of the resulting key
+
+# authType: http
+# endpointUrl: https://someurl.com/validateLogin
+# permittedStatusCodes: [ 200 ]
+# requestTimeout: 2000
```

Comment:
I would suggest to put the auth stuff into its own object, even as comment.
This makes really easy so switch between the types by just selecting the lines and use the __toggle comments__ feature

```yaml
#Authentication
auth:
  type: open

  # # or file type
  # type: file
  # # path to the user file
  # path: ./users.json 
  # # the name of a HMAC digest algorithm
  # hash: 'md5'
  # # the number of times the algorithm should be applied
  # iterations: 100 
  # # the length of the resulting key
  # keyLength: 32 
  # 
  # # or http type
  # type: http
  # endpointUrl: https://someurl.com/validateLogin
  # permittedStatusCodes: [ 200 ]
  # requestTimeout: 2000
```